DuckDuckGo/Suggestions/Model/SuggestionContainer.swift

```diff
@@ -52,8 +56,16 @@ final class SuggestionContainer {
         let urlFactory = { urlString in
             return URL.makeURL(fromSuggestionPhrase: urlString)
         }
-
-        self.init(suggestionLoading: SuggestionLoader(urlFactory: urlFactory),
+        let openTabsProvider: OpenTabsProvider = { @MainActor in
```

Comment:
Would using `WindowControllersManagerProtocol` make more sense here since other data are passed this way too?